console/backend/src/main/resources/FrankConsoleContext.xml

```diff
@@ -11,7 +11,7 @@
 
 	<context:property-placeholder />
 
-	<context:component-scan base-package="org.frankframework.management.web,org.frankframework.web">
+	<context:component-scan base-package="org.frankframework.management.web,org.frankframework.web, org.frankframework.management.web.socket">
```

Comment:
Dit zou niet moeten hoeven, hij pakt automatisch alle subfolders van `org.frankframework.management.web`

Reply:
Ah this was needed before a big merge from master where the file was deleted and remade, I copied this over to be sure it would still work